tests/framework/platform/fpga_hw.cpp

```diff
@@ -722,24 +755,28 @@ fpga_db *fpga_db::instance() {
 }
 
 static std::map<platform_cfg, std::string> platform_names = {
-  {  platform_cfg(0x8086, 0xbcc0, fpga_driver::linux_intel), "skx-p" },
-  {  platform_cfg(0x8086, 0xbcc1, fpga_driver::linux_intel), "skx-p-v" },
-  {  platform_cfg(0x8086, 0x09c4, fpga_driver::linux_intel), "dcp-rc" },
-  {  platform_cfg(0x8086, 0x09c5, fpga_driver::linux_intel), "dcp-rc-v" },
-  {  platform_cfg(0x8086, 0xbcc0, fpga_driver::linux_dfl0),  "skx-p-dfl0_patchset2" },
-  {  platform_cfg(0x8086, 0x09c4, fpga_driver::linux_dfl0),  "dcp-rc-dfl0_patchset2" },
-  {  platform_cfg(0x8086, 0x0b30, fpga_driver::linux_intel), "dcp-vc" },
-  {  platform_cfg(0x8086, 0x0b31, fpga_driver::linux_intel), "dcp-vc-v" },
-  {  platform_cfg(0x8086, 0x0b30, fpga_driver::linux_dfl0),  "dfl-n3000" },
-  {  platform_cfg(0x8086, 0x0b2b, fpga_driver::linux_dfl0),  "dfl-d5005" },
-  {  platform_cfg(0x8086, 0xbcce, fpga_driver::linux_dfl0),  "dfl-n6000" },
-  {  platform_cfg(0x8086, 0xbccf, fpga_driver::linux_dfl0),  "dfl-n6000" },
-  
+  { platform_cfg(0x8086, 0xbcc0, 0x8086, 0x0000, fpga_driver::linux_intel), "skx-p"                 },
+  { platform_cfg(0x8086, 0xbcc1, 0x8086, 0x0000, fpga_driver::linux_intel), "skx-p-v"               },
+  { platform_cfg(0x8086, 0x09c4, 0x8086, 0x0000, fpga_driver::linux_intel), "dcp-rc"                },
+  { platform_cfg(0x8086, 0x09c5, 0x8086, 0x0000, fpga_driver::linux_intel), "dcp-rc-v"              },
+  { platform_cfg(0x8086, 0xbcc0, 0x8086, 0x0000, fpga_driver::linux_dfl0),  "skx-p-dfl0_patchset2"  },
```

Comment:
can you remove skx-p-dfl0_patchset2 , dcp-rc-dfl0_patchset from table,

Reply:
That seems like an unrelated change.  We should do that in a different pull request.

Reply:
🐺